lib/firmata.js

```diff
@@ -562,8 +555,17 @@ var Board = function(port, options, callback) {
           SYSEX_RESPONSE[this.currentBuffer[1]] &&
           this.currentBuffer[this.currentBuffer.length - 1] === END_SYSEX) {
 
-          SYSEX_RESPONSE[this.currentBuffer[1]](this);
+          if (this.versionReceived) {
```

Comment:
I'm going to test this with a few different hardware setups before I comment.

- Arduino uno 
  - via Serial
  - via Ethernet 
- Leonardo 
  - via Serial 
  - via Ethernet (is that supported? I don't recall ever trying that)

That's just off the top, I have a lot of hardware here, so I'll see what I can come up with